mtp_noms_ops/apps/security/views/check.py

```diff
@@ -60,9 +64,45 @@ def get_context_data(self, **kwargs):
             {'name': self.title},
         ]
         context_data[self.object_context_key] = detail_object
-
+        context_data['related_credits'] = self._get_related_credits(api_session, context_data[self.object_context_key])
         return context_data
 
+    @staticmethod
+    def _get_related_credits(api_session, detail_object):
+        # Get the sender credits
+        sender_response = retrieve_all_pages_for_path(
+            api_session,
+            f'/senders/{detail_object["credit"]["sender_profile"]}/credits',
+            **{
+                'exclude_credit__in': detail_object['credit']['id'],
+                'check__isnull': False,
```

Comment:
i haven't tried running it, but is this the right filter name? the [api-594](https://github.com/ministryofjustice/money-to-prisoners-api/pull/594/) adds a filter called `security_check__isnull` rather than `check__isnull`. same for other the request below

Reply:
Haven't pushed the change which renames it in this PR yet